### PR TITLE
refactor(links): FRMW-736 Links migrated to Element

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-736

### LGTMs
- [x] Dev LGTM
- [x] Design LGTM

--- 
Refactor `Links` page to an `Element`.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `Links` refactored to `Element`

links - encoreui 2016-04-26 11-03-39

File: demo/scripts/demoApp.js [hunk 1]
```diff
@@ -344,17 +326,7 @@ angular.module('demoApp', ['encore.ui', 'ngRoute'])
...
-                            linkText: 'Links',
-                            href: '#/atoms/links'
-
-                        },
-                        {
-                            linkText: 'Lists',
-                            href: '#/atoms/lists'
-                        }
-                    ].concat(linksForModuleCategory('atoms'))
+                    children: linksForModuleCategory('atoms')
```
Comment (100stacks) [inline demo/scripts/demoApp.js]: Down to one `Atom` - `rxCheckbox`! 💯 

screen shot 2016-04-26 at 11 35 21 am

File: src/components/rxApp/common.less [hunk 2]
```diff
@@ -241,66 +241,6 @@ h6 {
   }
 }//.title
 
-// Statuses
```
Comment (100stacks) [inline src/components/rxApp/common.less]: Moved to `src/elements/Links/links.less`

**Comment by CITguy** [comment]
Added "Do Not Merge" because #1633 needs merged first.

**Comment by glynnis** [comment]
LGTM

**Comment by CITguy** [comment]
@100stacks, please rebase

**Comment by 100stacks** [comment]
@CITguy, ok rebased.

File: src/components/rxApp/common.less [hunk 3]
```diff
@@ -241,66 +241,6 @@ h6 {
...
-.link-action a:before {
-  content: "+ ";
-}
-
-.link-warn:before,
-.link-warn a:before {
-  content: "- ";
-}
-
-.action-separator {
```
Comment (CITguy) [inline src/components/rxApp/common.less]: marked as deprecated and moved below `.filter-box`

**Comment by CITguy** [comment]
LGTM

**Pull request merged**